apps/storefront/src/App.tsx

```
@@ -195,9 +196,16 @@ export default function App() {
       };
 
       if (!customerId) {
-        const info = await getCurrentCustomerInfo();
-        if (info) {
-          userInfo.role = info?.role;
+        try {
```

Right now any non‑`isCompanyError` failure from `getCurrentCustomerInfo` is silently swallowed and the flow just continues. Could we either rethrow / fail the login flow, or at least log/track those errors so we don’t end up with a partially initialised `userInfo` and no visibility into why `getCurrentCustomerInfo` failed?